### PR TITLE
unix: set uv_interface_address_t.netmask on zos

### PR DESCRIPTION
**Pull request by cjihrig**
This commit initializes the `netmask` field on zos.

In the interest of transparency, I have no way of testing this other than running it on the CI.

cc: @richardlau 

CI run with only unrelated Windows failures: https://ci.nodejs.org/view/libuv/job/libuv-test-commit/1036/

Also, comparing the `platform_output` test output for this CI run:

```
# uv_interface_addresses:
#   name: LOOPBACK6       TCPIP   
#   internal: 0
#   physical address: 00:00:00:00:00:00
#   address: ::1
#   netmask: none
#   name: OSDL            
#   internal: 0
#   physical address: 00:00:00:00:00:00
#   address: 148.100.36.133
#   netmask: 255.255.255.0
```

Compared to an older run:

```
# uv_interface_addresses:
#   name: LOOPBACK6       TCPIP   
#   internal: 0
#   physical address: 00:00:00:00:00:00
#   address: ::1
#   netmask: none
#   name: OSDL            
#   internal: 0
#   physical address: 00:00:00:00:00:00
#   address: 148.100.36.133
#   netmask: none
```

### CODE REVIEW

File: src/unix/os390.c [hunk 1]
```diff
@@ -530,10 +531,15 @@ int uv_interface_addresses(uv_interface_address_t** addresses, int* count) {
...
+  int r;
 
   /* get the ipv6 addresses first */
   uv_interface_address_t* addresses_v6;
-  uv__interface_addresses_v6(&addresses_v6, &count_v6);
+  r = uv__interface_addresses_v6(&addresses_v6, &count_v6);
+
+  if (r != 0) {
+    return r;
+  }
```
Comment (richardlau) [inline src/unix/os390.c]: I think we may want to continue to attempt to find IPv4 interfaces if `uv__interface_addresses_v6` fails, or at least in some of the failure cases. 

For example, according to https://www.ibm.com/support/knowledgecenter/en/SSLTBW_2.2.0/com.ibm.zos.v2r2.bpxbd00/ioctl.htm, `ioctl` with `SIOCGIFCONF6` can fail with:
> **EAFNOSUPPORT**
>    No IPv6 enabled TCP/IP stacks are active.

Comment (cjihrig) [reply src/unix/os390.c]: That's a good point. I can make this check more selective, but before we were ignoring *all* errors.

**Comment by richardlau** [comment]
@cjihrig I'm not sure who's in @libuv/zos (since I can't see the group), but perhaps you could @-mention them (since I can't). I can also reach out internally to the IBMers involved in porting libuv/Node.js over to z/OS when I'm in the office tomorrow.

**Comment by cjihrig** [comment]
@jBarz is the only member of libuv/zos.

**Comment by jBarz** [comment]
Hi, sorry for the delayed response. Since I no longer work at IBM, I don’t have access to a zos machine.
@richardlau: you should contact the IBMers and have them be members of the zos group.

**Comment by cjihrig** [comment]
cc @mhdawson maybe

**Comment by cjihrig** [comment]
Closing this. I'll let the IBM folks sort this out.

**Pull request closed without merging**